draft-ietf-quic-http.md

```diff
@@ -553,21 +554,12 @@ Different values for the same parameter can be advertised by each peer. For
 example, a client might permit a very large HPACK state table while a server
 chooses to use a small one to conserve memory.
 
-A SETTINGS frame MAY be sent at any time by either endpoint over the lifetime
-of the connection.
+Parameters are processed in the order in which they appear, and the value of a
+SETTINGS parameter is the last value that is seen by a receiver. The receiver of
+a SETTINGS frame does not need to maintain any state other than the last value
+of a given parameter.
```

Comment:
Can we forbid duplicates?  We wouldn't necessarily require enforcement, but it would avoid one of the nasty cases where there a bad actor can cause an unbounded amount of work to be performed (assuming here that skipping unsupported values is essentially free).